djangocms_form_builder/settings.py

```diff
@@ -29,6 +28,14 @@
 theme_render_path = f"{theme}.frameworks.{framework}"
 theme_forms_path = f"{theme}.forms"
 
+if not getattr(django_settings, 'DJANGO_FORM_BUILDER_SPACER_CHOICES', False):
+    if not getattr(django_settings, 'DJANGOCMS_FRONTEND_SPACER_SIZES', False):
+        SPACER_SIZE_CHOICES = DEFAULT_SPACER_SIZE_CHOICES
+    else:
+        SPACER_SIZE_CHOICES = django_settings.DJANGOCMS_FRONTEND_SPACER_SIZES
```

Comment:
I think `DJANGOCMS_FRONTEND_SPACER_SIZES` is essentially a list like `(("0", "..."), ("1", ...), ...) ` and needs to be converted to `(("mb-0", ...), ("mb-1", ...), ...)`.

Reply:
The "mb" at least for the Bootstrap framework.